# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmTlsData.h
# Commit: 25adf505426d
# Full Hash: 25adf505426d21f67e15d146f879b0ea53d343a7
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-03-23 04:09:13
# Regressor Bug: 1642412
# File Overlap Count: 19
# Description:
#   Bug 1642412 - wasm: Rollup of inline table.get/set for anyref. r=lth
#   
#   Rollup of the previous inline table.get/set patches.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141024
# ==============================================================================

diff -r 43f7556db2e1 -r 25adf505426d js/src/wasm/WasmTlsData.h
--- a/js/src/wasm/WasmTlsData.h	Tue Mar 22 19:44:13 2022 +0000
+++ b/js/src/wasm/WasmTlsData.h	Tue Mar 22 19:59:05 2022 +0000
@@ -73,7 +73,8 @@
 
   // Pointer to the array of elements (which can have various representations).
   // For tables of anyref this is null.
-  void* functionBase;
+  // For tables of functions, this is a pointer to the array of code pointers.
+  void* elements;
 };
 
 // Table element for TableRepr::Func which carries both the code pointer and
